core/connector/src/main/java/org/exoplatform/ecm/connector/dlp/FileDlpConnector.java

```diff
@@ -154,7 +159,8 @@ protected void treatItem(String entityId, Collection<SearchResult> searchResults
       Node node = session.getNodeByIdentifier(entityId);
       String fileName = node.getName();
       String restorePath = fixRestorePath(node.getPath());
-      if (!node.getPath().startsWith("/"+DLP_SECURITY_FOLDER+"/")) {
+      RestoredDlpItem restoredDlpItem = findRestoredDlpItem(node.getUUID());
+      if (!node.getPath().startsWith("/"+DLP_SECURITY_FOLDER+"/") && restoredDlpItem == null) {
```

Comment:
If fact, we should not check only null for restoredDlpItem.

If the object restoreDLPItem is null => treat it, no problem
If the object restoreDLPItem is not null : 
if the node.getModifiedDate is after restoreDlpItem.getDate : it means that the document was modified AFTER his restoration. So that it could be different from the one restored, so that, we must treat it.